components/browser/session/src/main/java/mozilla/components/browser/session/tab/CustomTabConfig.kt

```diff
@@ -75,6 +77,7 @@ data class CustomTabConfig(
         internal const val PAGE_TITLE_OPTION = "hasPageTitle"
         private const val MAX_CLOSE_BUTTON_SIZE_DP = 24
 
+        const val EXTRA_NAVIGATION_BAR_COLOR = "androidx.browser.customtabs.extra.NAVIGATION_BAR_COLOR"
```

Comment:
Sorry, forgot to mention this in the last pass: can you create an issue to remove this workaround when the new support library changes land?

Reply:
Filed #3963